pkg/api/versions.go

```diff
@@ -52,12 +52,19 @@ type Version struct {
 	RemovedIn string `json:"removed-in" yaml:"removed-in"`
 	// ReplacementAPI is the apiVersion that replaces the deprecated one
 	ReplacementAPI string `json:"replacement-api" yaml:"replacement-api"`
+	// Component is the component associated with this version
+	Component string `json:"-" yaml:"-"`
```

Comment:
This will make the list-versions command output the component which I think is important
```suggestion
	Component string `json:"component" yaml:"component"`
```